forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -75,17 +103,21 @@ def _exhaustive_dfe(program: Program, qubits: Sequence[int], in_states,
 
         if len(i_st) == 0:
             continue
+            # yield ExperimentSetting(
+            #     in_state=i_st, # need a don't care prep here
+            #     out_operator=PauliTerm(), # need an identity measurement
+            # )
```

Comment:
Can we remove these commented blocks? There are a couple

Reply:
Done